tests/components/nx584/test_binary_sensor.py

```diff
@@ -134,6 +135,7 @@ def test_sensor_normal(self):
         assert "foo" == sensor.name
         assert not sensor.should_poll
         assert sensor.is_on
+        assert sensor.device_state_attributes["zone_number"] == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5662368)